### ISSUE TITLE
Macro dont work on 3.x?

### ISSUE THREAD

**Issue by Ragash** [a2lix/TranslationFormBundle#318]
## Subject

Configuring "Fields" and divinding them in partial translations into formViews cause two behaviours unexpected:

1.  It renders all the configured fields of the translationType field, ignoring completly the command:
`            {{ a2lixTranslations.partialTranslations(traitForm.translations, ['title','description']) }}` 

if your TranslationType field is made of 4 configured fields, then you will be presented by 4 fields no matter what.

it seems the problem reside into the macro itself on line 34 and 35:
```
                {{ form_errors(translationsFields) }}
                {{ form_widget(translationsFields) }}
```
i suppose it is a Typo, cause those rows should be like this to mirror the "for" just above:

```
                {{ form_errors(translationsField) }}
                {{ form_row(translationsField) }}
```
also, using "form_widget" ignore all the options you could have setted, like translations labels, so rows seems to be more appropriate also.

2. as stated here: https://github.com/a2lix/TranslationFormBundle/pull/263 
there is a problem with boostrap 4.0 and 4.1 cause the sharing of ids in case of multiple partials on the same view

i suggested a solution for both problems, but probably it will need a better styling.
thanks!

**Comment by tchapi** [comment]
Hello @Ragash 

Apart from the `id` problems that should be resolved when the #263 PR is merged, are you still experiencing the first point you mention ? It seems that https://github.com/a2lix/TranslationFormBundle/commit/1fb343c9de9121f890a6f3a8dbde3ff820d2f9be#diff-65972ff017a34bf303247702fae62b5788f9be5fd30db3d70b83ca02aed3c17e may have corrected it

Thanks !

**Comment by tchapi** [comment]
Closing as it seems to be corrected already (see previous comment)
